_sass/_button.scss

```diff
@@ -1,105 +1,122 @@
 .button {
-    display: inline-block;
-    background: white;
-    border-radius: 4px;
-    padding: 10px 20px;
-    color: $teachla-green;
-    border: 2px solid $teachla-green;
-    text-decoration: none;
-    text-align: center;
-    font-weight: bold;
-    &:hover {
-        color: white;
-        background: $teachla-green;
-        cursor: pointer;
-    }
-
-    &:active {
-        background-color: $acm-black;
-        border: 2px solid $acm-black;
-    }
-    >.fa, .fab, .fas {
-        margin-right: 0.5em;
-    }
+  display: inline-block;
+  background: white;
+  border-radius: 4px;
+  padding: 10px 20px;
+  color: $teachla-green;
+  border: 2px solid $teachla-green;
+  text-decoration: none;
+  text-align: center;
+  font-weight: bold;
+  &:hover {
+    color: white;
+    background: $teachla-green;
+    cursor: pointer;
+  }
+
+  &:active {
+    background-color: $acm-black;
+    border: 2px solid $acm-black;
+  }
+  > .fa,
+  .fab,
+  .fas {
+    margin-right: 0.5em;
+  }
 }
 
 .button.button-sm {
-    padding: 4px 12px;
+  padding: 4px 12px;
 }
 
-.button-dark{
-    color: $acm-black;
-    border: 2px solid $acm-black;
-    &:hover {
-        color: $teachla-green;
-        background: $acm-black;
-        cursor: pointer;
-    }
+.button-dark {
+  color: $acm-black;
+  border: 2px solid $acm-black;
+  &:hover {
+    color: $teachla-green;
+    background: $acm-black;
+    cursor: pointer;
+  }
 
-    &:active {
-        background-color: white;
-        border: 2px solid $acm-black;
-    }
+  &:active {
+    background-color: white;
+    border: 2px solid $acm-black;
+  }
 }
 
-.button.button-ai{
+.button.button-ai {
+  color: $ai-blue;
+  border: 2px solid $ai-blue;
+  &:hover {
+    color: white;
+    background: $ai-blue;
+    cursor: pointer;
+  }
+
+  &:active {
     color: $ai-blue;
+    background-color: white;
     border: 2px solid $ai-blue;
-    &:hover {
-        color: white;
-        background: $ai-blue;
-        cursor: pointer;
-    }
-
-    &:active {
-        color: $ai-blue;
-        background-color: white;
-        border: 2px solid $ai-blue;
-    }
+  }
+}
+.button.button-rnative {
+  color: $rnative-blue;
+  border: 2px solid $rnative-blue;
+  &:hover {
+    color: white;
+    background: $rnative-blue;
+    cursor: pointer;
+  }
+
+  &:active {
+    color: $rnative-blue;
+    background-color: white;
+    border: 2px solid $rnative-blue;
+  }
 }
```

Comment:
Note: lots of code duplication. We should either use a mixin or property overrides!